packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx

```diff
@@ -0,0 +1,358 @@
+import * as React from 'react';
+import { useACMSafeFetch } from '@odf/mco/hooks/acm-safe-fetch';
+import { DRPlacementControlModel } from '@odf/mco/models';
+import { SearchResult } from '@odf/mco/types';
+import { MultiSelectDropdown } from '@odf/shared/dropdown/multiselectdropdown';
+import { SingleSelectDropdown } from '@odf/shared/dropdown/singleselectdropdown';
+import { getName, getNamespace } from '@odf/shared/selectors';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { AsyncLoader } from '@odf/shared/utils/AsyncLoader';
+import {
+  PairElementProps,
+  NameValueEditorPair,
+} from '@odf/shared/utils/NameValueEditor';
+import { ObjectReference } from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+import {
+  Alert,
+  AlertVariant,
+  Button,
+  Form,
+  FormGroup,
+  SelectOption,
+  SelectVariant,
+  Text,
+} from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { queryAppWorkloadPVCs } from '../utils/acm-utils';
+import { getClusterNamesFromPlacements } from '../utils/parser-utils';
+import { PlacementType, DRPlacementControlType } from '../utils/types';
+import '../../../../style.scss';
+import '../style.scss';
+
+const LABEL = 'label';
+const SPLIT_CHAR = '; ';
+
+const findPlacement = (placements: PlacementType[], name: string) =>
+  placements.find((placement) => getName(placement) === name);
+
+const getPlacementTags = (drpcs: DRPlacementControlType[]) =>
+  !!drpcs.length
+    ? drpcs.map(
+        (drpc) => [getName(drpc.placementInfo), drpc?.pvcSelector] || []
```

Comment:
u will never reach this condition...
```suggestion
        (drpc) => [getName(drpc.placementInfo), drpc?.pvcSelector]
```